electron/demokit/mouse.mm

```diff
@@ -103,8 +103,9 @@ void PerformClick(const Nan::FunctionCallbackInfo<v8::Value>& info)
             return;
         }
         
-        int x = info[0]->ToNumber()->Int32Value();
-        int y = info[1]->ToNumber()->Int32Value();
+        v8::Isolate* isolate = info.GetIsolate();
+        int x = info[0]->ToNumber(isolate)->Int32Value();
+        int y = info[1]->ToNumber(isolate)->Int32Value();
```

Comment:
I wonder if there's a way to use nan in the future to avoid having to keep updating this for every version of v8. On the other hand, may be overkill since its not meant to compile on anything other than latest.

Reply:
👍